@ecomplus/storefront-components/src/html/ProductVariations.html

```diff
@@ -6,7 +6,23 @@
     v-for="(options, grid, index) in variationsGrids"
     :class="`variations__grid variations__grid--${grid}`"
   >
-    <h5>{{ getGridTitle(grid) }}</h5>
+  <div
+    class="variations__grid__title"
+  >
```

Comment:
```suggestion
  <div class="variations__grid-title">
```

As quebras de linha nos elementos são seguindo o style guide do Vue https://vuejs.org/v2/style-guide/#Multi-attribute-elements-strongly-recommended
Mas seria pra componentes com mais de um atributo, neste caso facilita a visualização, quando é um atributo só não tem necessidade...

`variations__grid__title` => `variations__grid-title`
Nos componentes estou convencionando um nível só do BEM, pra evitar poluição demais